pkg/diagnose/cni.go

```diff
@@ -19,18 +19,30 @@ limitations under the License.
 package diagnose
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"strings"
 
+	"github.com/mcuadros/go-version"
 	"github.com/pkg/errors"
 	"github.com/submariner-io/admiral/pkg/reporter"
 	"github.com/submariner-io/subctl/pkg/cluster"
 	submv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
 	"github.com/submariner-io/submariner/pkg/cni"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+const (
+	ovnKubeDBPodLabel = "ovn-db-pod=true"
```

Comment:
I'm not sure if we use the same labels downstream since all ovn-k containers are in a pod

Reply:
Oh maybe it will
https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/managed/ovnkube-master.yaml#L54

Just be cautious in the OCP deployment nbdb sbdb are just containers running in the same pod

Reply:
Yeah, saw that. What matters for us is if it can run ovn-nbctl and that works in both containers.